src/zenml/models/flavor_models.py

```diff
@@ -11,27 +11,28 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
-"""Model definitions for stack component flavors."""
+"""Models representing stack component flavors."""
 
-from typing import ClassVar, List, Optional
+from typing import Optional
 
-from pydantic import Field
+from pydantic import BaseModel, Field
 
 from zenml.enums import StackComponentType
-from zenml.models.base_models import ProjectScopedDomainModel
-from zenml.utils.analytics_utils import AnalyticsTrackedModelMixin
+from zenml.models.base_models import (
+    ProjectScopedRequestModel,
+    ProjectScopedResponseModel,
+)
+from zenml.models.constants import MODEL_CONFIG_SCHEMA_MAX_LENGTH
 
+# TODO: Add example schemas and analytics fields
 
-class FlavorModel(ProjectScopedDomainModel, AnalyticsTrackedModelMixin):
-    """Domain model representing the custom implementation of a flavor."""
+# ---- #
+# BASE #
+# ---- #
 
-    ANALYTICS_FIELDS: ClassVar[List[str]] = [
-        "id",
-        "type",
-        "integration",
-        "project",
-        "user",
-    ]
```

Comment:
These are missing in the new models and are an absolute must for our analytics. I don't know if you can find a way to "merge" these dynamically into the fields that come from the inherited classes, but that would probably be the best way to do it.